eva-tools/src/main/java/embl/ebi/variation/eva/vcfdump/VariantExporterController.java

```diff
@@ -107,21 +108,34 @@ public QueryOptions getQuery(MultivaluedMap<String, String> queryParameters) {
 //        return new VariantExporter(cellBaseClient, variantDBAdaptor, query).vcfExport(outputDir, variantSourceDBAdaptor);
 //    }
 
-    public void run() throws IOException {
+    public void run() {
+        Map<String, VCFHeader> headers = getVcfHeaders();
+        if (headers != null) {
```

Comment:
I think an Exception should be thrown if `headers` is null, the application won't work properly without them.